frontend/src/components/parties/PartyProfile.js

```diff
@@ -63,6 +79,46 @@ export class PartyProfile extends Component {
     });
   }
 
+  openEditPartyModal = (event, party, onSubmit, title, isPerson, provinceOptions) => {
+    const initialValues = {
+      email: party.email && party.email !== "Unknown" ? party.email : null,
+      first_name: party.first_name ? party.first_name : null,
+      party_name: party.party_name ? party.party_name : null,
+      phone_ext: party.phone_ext ? party.phone_ext : null,
+      phone_no: party.phone_no ? party.phone_no : null,
+      address_line_1:
+        party.address[0] && party.address[0].address_line_1
```

Comment:
You could pull `party.address[0]` out into a variable above `initialValues` to avoid repeating that every time.

Or... it looks like every address property includes a check for `party.address[0]`. A lot of this duplication could be removed with something like:
```javascript
// If the explicit nulls aren't needed:
const addressValues = party.address[0] || {}

// OR If the explicit nulls are needed:
const partyAddress = party.address[0];
const addressValues = !partyAddress ? {} : {
  address_line_1: partyAddress.address_line_1 || null,
  // the rest
}

// Regardless...
const initialValues = {
  email: party.email && party.email !== "Unknown" ? party.email : null,
  // other stuff
  ...addressValues
};
```

Reply:
Updated. Looks like party.address[0] is always returned. Removed that check from all the address fields.